queries/gdscript/highlights.scm

```diff
@@ -20,7 +20,7 @@
   (name) @type) @keyword
 
 (const_statement
-  (name) @constant) @keyword
+  (name) @constant) @type.qualifier
```

Comment:
is it really the right thing to highlight the *whole* statement as keyword (I know it was that way before)